saleor/discount/utils.py

```diff
@@ -64,8 +64,8 @@ def release_voucher_usage(order_data: dict):
     voucher = order_data.get("voucher")
     if voucher and voucher.usage_limit:
```

Comment:
What do you think about this? 👇  A littile bit more lines but you don't repease a condition `if voucher` 
```
def release_voucher_usage(order_data: dict):
    if not voucher:
        return
    if voucher.usage_limit:
```

Reply:
yes, I think it's more elegant this way. fixed it.